sw/airborne/subsystems/radio_control/sbus_common.c

```diff
@@ -54,6 +54,124 @@
 #endif
 
 
+
+/*
+ * S.bus decoder matrix. (extracted from PX4)
```

Comment:
Can you provide instead some pointer on the SBUS2 protocol ? It looks hardly possible that this changes won't affect the support of SBUS1.
As a side not, an other pull request is also changing some elements of the decoding function (#2018), but only for efficiency, no extra protocol support.